integration/compose/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.bumptech.glide.integration.compose">
-  <uses-sdk android:minSdkVersion="21" />
```

Comment:
@kanelbulle can you try importing this PR and see if it builds? I think these min sdk versions are used internally, but I could be wrong.